src/adv.py

```diff
@@ -33,19 +35,47 @@
 room['narrow'].n_to = room['treasure']
 room['treasure'].s_to = room['narrow']
 
-#
-# Main
-#
-
-# Make a new player object that is currently in the 'outside' room.
-
-# Write a loop that:
-#
-# * Prints the current room name
-# * Prints the current description (the textwrap module might be useful here).
-# * Waits for user input and decides what to do.
-#
-# If the user enters a cardinal direction, attempt to move to the room there.
-# Print an error message if the movement isn't allowed.
-#
-# If the user enters "q", quit the game.
+# define items
+rock = Item("Rock", "heavy rock for smashing toes.")
+sword = Item("Sword", "stricketh thee with thy sword")
+oracle = Item("oracle", "Life precious mysteries revealed")
+wand = Item(
+    "Wand", "Swaths of snakes approach to battle you")
+
+# place items in rooms
+room['outside'].items.append(rock)
+room['outside'].items.append(oracle)
+room['foyer'].items.append(wand)
+room['narrow'].items.append(sword)
+
+# welcome player - learn player's name
+print('Welcome to your fated destiny player!')
+player_input = input('What is your name?: ')
+player = Player(player_input, room['outside'])
+print(f'Well, {player.name} get ready for the adventure of a lifetime.')
+
+# print players current room
+current_room = player.current_room
+print(current_room)
+
+# define valid directions
+valid_directions = ["n", "s", "e", "w"]
+
+while True:
+    # Wait for user input
+    cmd = input(
+        "Please enter a cardinal direction in the form of 'n', 's', 'e', or 'w' \n or press 'i' to see your inventory \n or press 'q' to quit -> ")
+    # Parse user inputs (n, s, e, w, q)
+    if cmd in valid_directions:
+        # If input is valid, move the player and loop
+        player.travel(cmd)
+        if len(current_room.items) > 0:
+            player.items.extend(current_room.items)
```

Comment:
I might be wrong but doesn't this automatically add items to the player's `items` when entering the room? Also every time the player enters the room, they get the duplicated items again?